### PR TITLE
Refactor i18n files structure

### PR DESCRIPTION
**Pull request by amed**
Make Deutsch version is the default to be shown in index page

### CODE REVIEW

**Comment by jhilden** [comment]
Deploy preview ready!

Built with commit 3ebc28b85ed98341c165c361d80b3e75541114ad

https://deploy-preview-40--webmontag.netlify.com

File: source/layouts/events-all.html.slim [hunk 1]
```diff
@@ -17,7 +17,7 @@ html.no-js
             = partial "shared/content/_event_box.slim",
               locals: {locale: 'de', next_event: next_event, upcoming_events: upcoming_events}
           #event-data
-            = partial "shared/content/_archive.slim", locals: {past_events: past_events, date_format: :day_month}
+            = partial "shared/content/_archive.slim", locals: {past_events: past_events, locale: 'de', date_format: :day_month}
```
Comment (jhilden) [inline source/layouts/events-all.html.slim]: hash wants to breathe

File: source/shared/content/_archive.slim [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+h2
+  =I18n.t('events.past.title', locale: locale)
 -events_grouped_by_year = past_events.group_by{ |event| event.date[0,4] }
 -events_grouped_by_year.each do |year, events|
   .year-wrap
     h4
       ="#{year}"
     ul
-      -events.each do |event|
+      -events.reverse.each do |event|
```
Comment (jhilden) [inline source/shared/content/_archive.slim:9]: write a spec for this bugfix

**Review by jhilden — CHANGES_REQUESTED** [review]

File: source/shared/content/_archive.slim (hunk 2 above)
Comment (amed) [reply source/shared/content/_archive.slim:9]: https://github.com/webmontag/webmontag-koeln.de-reloaded/issues/41

[bug] Write spec for month arrangement.

**Pull request merged**